packages/Debian/common/DEBIAN/postrm

```diff
@@ -35,8 +35,8 @@ remove_fledge_sudoer_file() {
     rm -rf /etc/sudoers.d/fledge
 }
 
-echo "Cleanup of files"
-remove_unused_files
+#echo "Cleanup of files"
+#remove_unused_files
```

Comment:
Revert the changes as upgrade handled changes made into preinst. 
And this function improvisation will be via FOGL-3424

Reply:
I think we need this, as an upgrade is never  possible having remove_unused_files.

than eventually a following task will apply a different solution.

Reply:
I would suggest you better to completely remove only the function call from here instead of commented code lines.
And function definition stay as it is; We have the FOGL-3424 to handle this situation in same function.

Reply:
done